packages/screens/LaunchpadERC20/hooks/useCreateSale.tsx

```diff
@@ -0,0 +1,95 @@
+import { useRoute } from "@react-navigation/native";
+import { useMemo } from "react";
+import { z } from "zod";
+import { create } from "zustand";
+
+import { useAppNavigation } from "@/utils/navigation";
+
+export const zodCreateSaleForm = z.object({
```

Comment:
Please put the zod stuff and types in a types.ts file. Maybe in `LaunchpadERC20/utils/types.ts`

Reply:
[bff1a7d](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/bff1a7d69ebe904409cc45f68221626653c1aaec)